doc/docs/master/reference/clients.md

```diff
@@ -25,6 +25,19 @@ cd $GOPATH/src/google.golang.org/grpc
 git checkout 21f8ed309495401e6fd79b3a9fd549582aed1b4c
 ```
 
+### Running Go Examples
+
+The Pachyder, [godocs](https://godoc.org/github.com/pachyderm/pachyderm/src/client) reference
```

Comment:
fix typo:

The Pachyderm, [godocs](https://godoc.org/github.com/pachyderm/pachyderm/src/client) reference